lib/routes.js.coffee

```diff
@@ -34,7 +34,7 @@ Utils =
           s.push "#{encodeURIComponent(prefix.toString())}=#{encodeURIComponent(object.toString())}"
 
     return "" unless s.length
-    s.join("&")
+    (item for item in s when item != "").join("&")
```

Comment:
```Because the == operator frequently causes undesirable coercion, is intransitive, and has a different meaning than in other languages, CoffeeScript compiles == into ===, and != into !==. In addition, is compiles into ===, and isnt into !==.```

http://coffeescript.org/#operators